src/pages/main-page/OurProjects.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styles from './assets/our-projects.module.scss';
+import background from '../../assets/ourProjectsBackground.png';
+import RocketPanel from '../../components/rocket-panel/RocketPanel';
+import r1 from '../../assets/rockets/r1.png';
+import r2 from '../../assets/rockets/r2.png';
+import r3 from '../../assets/rockets/r3.png';
+import r4 from '../../assets/rockets/r4.png';
+import r4v2 from '../../assets/rockets/r4v2.png';
+
+const rockets = [
+    'R1',
+    'R2',
+    'R3',
+    'R2S',
+    'R4S',
+    'R4',
+    'R4V2',
+    'R5'
+];
+
+export default function OurProjects() {
+    const [selectedButton, setSelectedButton] = React.useState<string>('R4V2');
+
+    const handleButtonChange = (rocket) => {
+        if (rocket === selectedButton) {
+            return styles.sectionButtonSelected;
+        } else {
+            return styles.sectionButton;
+        }
+    };
+
+    const createButtons = () => {
+        return rockets.map(rocket => {
+            return (
+                <button className={handleButtonChange(rocket)} onClick={() => setSelectedButton(rocket)}>
+                    {rocket}
+                </button>
+            );
+        });
+    };
+
+    const chooseRocketPanel = () => {
+        switch (selectedButton) {
```

Comment:
it is better to have here list or map and search in it instead of switch case. Then you can also use its keys or prop instead of list of rockets at the beginning